boards/xtensa/yd_esp32/Kconfig.board

```diff
@@ -0,0 +1,8 @@
+# YD-ESP32 board configuration
+
+# Copyright (c) 2023 Julio Cesar <hi@jcsx.dev>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_ESP32
+	bool "YD-ESP32 Development Board"
+	depends on SOC_ESP32
```

Comment:
this needs to be altered to `CONFIG_SOC_SERIES_ESP32` as the `CONFIG_SOC_ESP32` is no longer available on the current upstream

Reply:
Done.

https://github.com/zephyrproject-rtos/zephyr/blob/522c80b961e0fc1f9cdf8e458dc58d34e7859227/boards/xtensa/yd_esp32/Kconfig.board#L5-L8